packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content/faq-content.tsx

```diff
@@ -83,32 +87,30 @@ const FAQContent = observer(({ faq_list, hide_header = false }: TFAQContent) =>
                     {title}
                 </Text>
             ),
-            content: description.map((item, index) => (
+            content: description?.map((item, index) => (
                 <FAQ {...item} key={`faq-description-item-${item?.content}-${index}`} />
             )),
         }));
     };
 
     return (
-        <div>
+        <div data-testid='id-faq__wrapper'>
             <div className='faq__wrapper' ref={faq_wrapper_element}>
-                {!hide_header && (
-                    <Text as='p' line_height='xl' className='faq__wrapper__header' weight='bold'>
-                        {localize('FAQ')}
-                    </Text>
-                )}
-                {faq_list?.length ? (
-                    <div onClick={handleAccordionClick}>
-                        <Accordion className='faq__wrapper__content' list={getList()} icon_close='' icon_open='' />
-                    </div>
-                ) : (
-                    <div className='faq__wrapper__nosearch'>
-                        <Text as='h1' weight='bold' line_height='xxs'>
-                            {localize('No results found "{{ faq_search_value }}"', {
-                                faq_search_value,
-                            })}
-                        </Text>
-                    </div>
+                {faq_list && faq_list?.length > 0 && (
```

Comment:
Here I think `faq_list?.length` should be enough for the check

Reply:
done ✅